evap/student/views.py

```diff
@@ -105,12 +105,51 @@ def sorter(evaluation):
         )
 
     unfinished_evaluations.sort(key=sorter)
+    info = Evaluation.annotate_with_participant_and_voter_counts(
+        Semester.active_semester()
+        .evaluations.filter(is_single_result=False)
+        .exclude(state__lt=Evaluation.State.APPROVED)
+        .exclude(is_rewarded=False)
+        .exclude(id__in=settings.EXCLUDED_EVALUATION_IDS)
+        .exclude(course__type__id__in=settings.EXCLUDED_COURSE_TYPE_IDS)
+        .exclude(course__is_private=True)
+    ).aggregate(
+        total_votes=Sum("num_voters"),
+        participants=Sum("num_participants"),
+    )
+    if settings.GLOBAL_EVALUATION_PROGRESS_REWARDS:
+        max_reward = max(settings.GLOBAL_EVALUATION_PROGRESS_REWARDS)
```

Comment:
Maybe
```suggestion
        max_reward_participation_quote, max_reward_value = max(settings.GLOBAL_EVALUATION_PROGRESS_REWARDS)
```

Or maybe we should use a dataclass / namedtuple thing in the settings, so we can have a `reward` instance that has `.participation_ratio` and `.value` or `.text` or something. The disadvantage of unpacking here is that the order of the definition matters and switching the order breaks things in weird ways.

Reply:
I'd just keep the simple version here. For me, adding a comment that the order is important is okay for now. In the long run, the settings will probably be replaced by proper database objects anyway.